models/issue_label.go

```diff
@@ -71,6 +71,7 @@ type Label struct {
 	IsChecked       bool `xorm:"-"`
 	QueryString     string
 	IsSelected      bool
+	IsExcluded      bool
```

Comment:
Why we need this new column?

Reply:
Its for show blocked icon

https://github.com/jaqra/gitea/blob/34c503c5573995ad0df4803df86647341b0d5de7/templates/repo/issue/list.tmpl#L53

Reply:
So this will be shared by all the users of this repository?

Reply:
I made it same with `IsSelected`

I dont know if its correct

Reply:
@jaqra This kind of value is not necessarily tied to the model. Perhaps it is better placed in the request context. I didn't check your code, but if the column is transient, you should at least use `xorm:="-"` to avoid adding it to the database.

Reply:
Added `xorm:="-"`

Reply:
On this point, and not necessarily for your PR, why is IsSelected not `xorm:"-"` too. I can't see why we would want that persisted.

Reply:
@zeripath those two lines (`QueryString` and `IsSelected`) added #5786 as rework of #3438. Parhaps no one did not see that ?

Reply:
Sounds like an error to me. As @lunny 've said, those fields are related to one user's session and should not be affected by other user's work (perhaps nobody have noticed because the value from the database is not actually used?)

Reply:
We should send another PR to add xorm ignore tag on the fields #5786 added wrong.

Reply:
> We should

@lunny done in #8633